islandora_rdm_disk_usage/src/Controller/DiskUsageController.php

```diff
@@ -11,226 +11,242 @@
  */
 class DiskUsageController extends ControllerBase {
 
-    /**
-     * An instance variable.
-     *
-     * @var entityTypeManager
-     */
-    protected $entityTypeManager;
-
-    /**
-     * Constructs a new DiskUsageController object.
-     */
-    public function __construct(EntityTypeManagerInterface $manager) {
-        $this->entityTypeManager = $manager;
+  /**
+   * An instance variable.
+   *
+   * @var entityTypeManager
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Constructs a new DiskUsageController object.
+   */
+  public function __construct(EntityTypeManagerInterface $manager) {
+    $this->entityTypeManager = $manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+              $container->get('entity_type.manager')
+      );
+  }
+
+  /**
+   * To create a report and it returns a render array.
+   */
+  public function createReports() {
+
+    $report['chart']['pageTitle'] = [
+      '#type' => 'markup',
+      '#markup' => '<h3>Charts</h3>',
+    ];
+    $report['chart']['content'] = [
+      '#type' => 'table',
+      '#header' => [
+        t('Group by Type'),
+        t('Group by Owner'),
+      ],
+    ];
+
+    // $rows[] = ["2222","444"];
+    /*$rows[] = [['data' => $this->createChart("bar", "type")],
+    ['data' => $this->createChart("bar", "owner")],
+    ];*/
+    $rows[] = [['data' => $this->createChart("pie", "type")],
+          ['data' => $this->createChart("pie", "owner")],
+    ];
+
+    $report['chart']['content']['#rows'] = $rows;
+    /*$report['chart']['button'] = array(
+    '#type' => 'button',
+    '#value' => t('switch chart'),
+    '#weight' => 19,
+    '#attributes' => ['id' => 'chart-toggle-button'],
+    '#attached' => array(
+    'library' => array('islandora_rdm_disk_usage/switch-button',),
+    ),
+    //$report['#attached']['library'][] = 'islandora_rdm_disk_usage/switch-button';
+    );*/
+    // \Kint::$maxLevels = 0;
+    $report['table'] = $this->createTable();
+
+    // Attach js file.
+    return $report;
+  }
+
+  /**
+   * Create a table with the entities. Using Render arrays.
+   */
+  private function createTable() {
+    $entities = $this->getMyList();
+
+    $report['pageTitle'] = [
+      '#type' => 'markup',
+      '#markup' => '<h3>Media Items List</h3>',
```

Comment:
Put the text inside of a $this->t() function, but keep the markup outside of it, like 

'<h3>' . $this->t('Media Items List') . '</h3>';